### DISCUSSION TITLE
Why meta data doesnt show on social media

### DISCUSSION THREAD

**Discussion by vertexgren** [nuxt/nuxt#24094] [Questions]
I`m generate my application with nuxt generate, and deploy my application on s3 bucket like SPA, on my pages i have useServerSeoMeta,
but my open graph properties doesn't show in social media, but if i run dev inspector on opened website i see my metatags
I have configured fallback to 200.html and when I forward the link to the site, it's like I'm forwarding 200.html but when you click on the link, the correct page with meta tags is displayed

**Comment by TheAlexLichter** [comment]
If you render with `ssr: false`, social networks won't be able to crawl the social media tags. Same goes for non-generated "fallback pages" as they are rendered SPA-style.

**Reply by itsmnthn** [reply]
Is there a way to pre-render only meta tags or selective meta tags even with `ssr: fase`. There are many use cases where server-side rendering is not needed but SEO is needed, I feel SEO is always needed.

I feel this should be a part of Nuxt3 Handling SEO with `ssr: false`

**Reply by vertexgren** [reply]
i render with `ssr: true`,but I think it's because I put the app in s3 bucket,

**Reply by TheAlexLichter** [reply]
> Is there a way to pre-render only meta tags or selective meta tags even with ssr: false.

Not with `ssr: false`, but:

You can use `nuxt generate` + `<ClientOnly>` in your `app.vue` (or children) - after setting your tags

Be aware that only the then prerendered pages have the OG tags set in the HTML.

**Reply by Picturk** [reply]
I couldn't get the facebook crawler to read my og:image. I had defined og:image in the meta properties in the nuxt.config.ts. On the pages components I tried and failed to use the useHead() composable to overwrite the og:image property in the nuxt.config.ts. Removing the meta property from the nuxt.config.ts worked for me...I ended up using the useSeoMeta() composable. The fix worked for the content module too. I defined my og:image using frontMatter in my .md files.

step 1. comment out 'meta' property from nuxt.config.ts

```
app: {
    htmlAttrs: { lang: 'en' },
    head: {
      // title: 'Upload Photos for the Happy Couple',
      /*
      meta: [
        {
          property: 'og:image',
          content:
            'https://weddingpixie.com/img/share/weddingPixie-1200x630-new.jpg',
          },
      ], 
     */
...
```

step 2. useSeoMeta() on your page component index.vue

```
useSeoMeta({
  title: ()=> uploadPageTitle.value,
  ogImage: 'https://weddingpixie.com/img/share/WeddingPixieUpload.png',
})
```
